### ISSUE TITLE
can't replicate the results in this paper

### ISSUE THREAD

**Issue by egrcc** [iesl/dilated-cnn-ner#1]
Hi,

I ran this code according to the instructions, but I can't replicate the results in this paper. The F1 score I got is 89.23. Could you tell me how can I replicate the results? The following is the steps I ran this code.

1. I copy the conll 2003 dataset from https://github.com/synalp/NER/tree/master/corpus/CoNLL-2003, and place the files in data/conll2003.
![_20171122225343](https://user-images.githubusercontent.com/6469953/33133581-39c5f498-cfd8-11e7-8775-13b25eb5d13c.png)

2. Set up environment variables.
```
export DILATED_CNN_NER_ROOT=`pwd`
export DATA_DIR=data
```

3. Get the Glove word embeddings from http://nlp.stanford.edu/data/glove.6B.zip, and use the 100d vector "glove.6B.100d.txt". The file is renamed  "lample-embeddings-pre.txt" and I place the file in data/embeddings.
![_20171122225359](https://user-images.githubusercontent.com/6469953/33133849-06e687bc-cfd9-11e7-8c33-5379ff066e8a.png)

4. Perform all data preprocessing.
```
./bin/preprocess.sh conf/conll/dilated-cnn.conf
```

5. Train a tagger.
```
./bin/train-cnn.sh conf/conll/dilated-cnn.conf
```

6. The output of the final iteration is:
![_20171122230513](https://user-images.githubusercontent.com/6469953/33134077-b7d04d2e-cfd9-11e7-9fc6-687497cbae8f.png)

7.  Evaluate a saved model on the test set:
```
./bin/eval-cnn.sh conf/conll/dilated-cnn.conf --load_model /home/ljzhao/dilated-cnn-ner-master/models/dilated-cnn.tf test
```

8. The output is:
![_20171122230823](https://user-images.githubusercontent.com/6469953/33134211-13970a9e-cfda-11e7-91e1-15ce95bc116c.png)

**Comment by strubell** [comment]
As we describe in the paper, we use the same skip-n-gram embeddings as
Lample et al. (2016), not Glove embeddings (I linked to the Glove
embeddings as they are more readily accessible). I've uploaded the
pre-trained embeddings here:
https://drive.google.com/open?id=1AQX_RGIc7Sopyi3cGEYBfbKV848VXzJn

Let me know if you still have issues with these embeddings. It's a holiday
right now in the US, but I can update the readme to use these embeddings
when I get a chance.
